source/_components/lupusec.markdown

```diff
@@ -0,0 +1,22 @@
+---
+layout: page
+title: "Lupus Electronics Home Security"
+description: "Instructions on integrating Lupusec home security with Home Assistant."
+date: 2018-10-03 16:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: lupusec.jpg
+ha_category: Hub
+ha_release: 0.79
```

Comment:
`0.79` has been shipped already some time ago. The next possible release would be `0.81`.